examples/io/plot_objects_from_arrays.py

```diff
@@ -148,7 +148,7 @@
 for ai, asig in enumerate(seg.analogsignals):
     # Since the data does not contain channel names, channel indices are used.
     ch_names.append('Neo %02d' % (ai + 1,))
-    data.append(asig.rescale('V').magnitude[:, 0])
+    data.append(asig.rescale('V').magnitude)
```

Comment:
... in other words, this will *undo* my fix. I think the version I added works for Neo 0.5.0 (latest) and your change here will work for older versions (what we had before)